helloworld/lib/greet.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+module.exports = function greet(name){
+  if (!name) throw new Error('expected a name');
```

Comment:
Read Feature tasks carefully. Remember the instructions said to check if the arg is a string, and your if statement doesn't do that.